README.rst

```diff
@@ -31,6 +31,31 @@ To install from source on github_::
 
 .. _github: https://github.com/chapel-lang/sphinxcontrib-chapeldomain
 
+Release
+-------
+
+#. Test and commit changes
+#. Merge your changes
+#. Go to main page for repo
+#. Click “Releases” on right side of screen
+#. Click “Draft a new release” button on top of screen
+#. For the tag, make a new tag with the new version number
+#. You can generate the release notes via the “generate release notes” button,
+   comparing against the most recent release.  This will autofill in the details
+   for you
+#. Click “Publish release”
+    - This will trigger the workflow to push a new release to PyPI, assuming no
+      problems have snuck into our release procedure since the last time it was
+      run
+#. Open a PR bumping the version to the next version number so that we’re ready
+   for the next change.  This should always be the first PR in a new release
+   (otherwise we’ll have build issues), so it should be straight-forward to
+   figure out how to do this
+
+IF SOMETHING GOES WRONG
+
+The release pushing job is in .github/workflows/python-publish.yml
```

Comment:
Rather than the all caps "if something goes wrong", consider a note like "the Github Action for publishing a new version lives in .github/workflows/python-publish.yml"